### PR TITLE
Add Dynamic Wallets Config Vars

### PR DESCRIPTION
**Pull request by sparkplug0025**
Solves #143

### CODE REVIEW

File: packages/webapp/src/_app/eos/ual/EdenUALProvider.tsx [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+const authenticators: any[] = [];
+if (availableWallets.indexOf("ANCHOR") >= 0) {
+    authenticators.push(anchor);
+}
+if (availableWallets.indexOf("SCATTER") >= 0) {
+    authenticators.push(scatter);
+}
+if (availableWallets.indexOf("LEDGER") >= 0) {
+    authenticators.push(ledger);
+}
```
Comment (brandonfancher) [inline packages/webapp/src/_app/eos/ual/EdenUALProvider.tsx:15]: `availableWallets.includes("ANCHOR")` is more readable here, IMHO.

**Review by brandonfancher — APPROVED** [review]
A minor suggestion, but good either way! I like how we're leveraging `.env.local` to get around the local Scatter issue. Nice!

**Comment by brandonfancher** [comment]
@sparkplug0025 Do you know if we need to add this new env var to Vercel, or will it just read it from the `.env` file?

File: packages/webapp/src/_app/eos/ual/EdenUALProvider.tsx (hunk 1 above)
Comment (sparkplug0025) [reply packages/webapp/src/_app/eos/ual/EdenUALProvider.tsx:15]: oooo `.includes`! nice!

**Comment by sparkplug0025** [comment]
> @sparkplug0025 Do you know if we need to add this new env var to Vercel, or will it just read it from the `.env` file?

I added already yesterday!

**Pull request merged**